notify-send.sh

```diff
@@ -122,6 +123,11 @@ notify() {
         echo "$NOTIFICATION_ID"
     fi
 
+    if [[ -n "$FORCE_EXPIRE" ]] ; then
+        SLEEP_TIME="$(bc <<< "scale=3; $EXPIRE_TIME / 1000")"
```

Comment:
I'm not fully sure  bc is installed on all systems where notify-send.sh is used and I prefer avoiding unnecessary hard dependency, soft dependcy is ok to me though. So I just would add a run-time check and print a friendly error message about bc not found, something like this:

```type bc &> /dev/null || { echo "bc command not found. Please installed bc package."; exit 1;  }```